### DISCUSSION TITLE
Chat

### DISCUSSION THREAD

**Discussion by JoshuaDaCosta** [JoshuaDaCosta/IMAG#4] [General]
@MANUEL-CARLOS

**Comment by JoshuaDaCosta** [comment]
Oiee

**Comment by Manuel-Carlos** [comment]
ouu é aqui?

**Reply by JoshuaDaCosta** [reply]
Vou te ofender rsrsrs

**Reply by JoshuaDaCosta** [reply]
Ainda não a aceitaste o convite bem feio vai nas tuas notificações

**Comment by Manuel-Carlos** [comment]
wey vc está a falar inglês porq? kk

**Reply by JoshuaDaCosta** [reply]
xeie

**Reply by JoshuaDaCosta** [reply]
voce colocou google tradutor fazer como??

**Comment by Manuel-Carlos** [comment]
oh desgraça

**Reply by JoshuaDaCosta** [reply]
Mano esqueci estava a fazer algo bro

**Comment by JoshuaDaCosta** [comment]
Wey meu wtp no tel parou de funcionar, nao me veres no wtp porque talvez saiu do pc

**Comment by Manuel-Carlos** [comment]
oh rapaz

**Reply by JoshuaDaCosta** [reply]
Fala aí bem feio

**Comment by JoshuaDaCosta** [comment]
Mano não esqueci temos que falar aqui ! @Manuel-Carlos

**Comment by Manuel-Carlos** [comment]
yh, já estou dentro, depois manda os mambos estilizados

**Reply by JoshuaDaCosta** [reply]
Já estilizei vou mandar hj mesmo as 21hrs

**Comment by JoshuaDaCosta** [comment]
Mn os da pesquisa de campos já foram fazer?

**Comment by JoshuaDaCosta** [comment]
![image](https://github.com/JoshuaDaCosta/IMAG/assets/152827574/a3740a89-107e-4a29-9d89-b3ccf85183d5)
OQ ACHASTE DESSE DESIGN?

**Reply by Manuel-Carlos** [reply]
está fix, mas as escritas IMAG troca de lugar com os links ENTRAR e  CRIAR. a cor dos textos se fossem um pouco mais pretas sairia bem

**Reply by JoshuaDaCosta** [reply]
daquele jeito ta fx vamos usar o teu estilo
